ui/app/components/secure-variable-form.js

```diff
@@ -110,13 +152,75 @@ export default class SecureVariableFormComponent extends Component {
     }
   }
 
+  //#region JSON Editing
+
+  view = this.args.view;
+  // Prevent duplicate onUpdate events when @view is set to its already-existing value,
+  // which happens because parent's queryParams and toggle button both resolve independently.
+  @action onViewChange([view]) {
+    if (view !== this.view) {
+      set(this, 'view', view);
```

Comment:
`nit` `non-blocking`: `view` is an enum that can be `table` or `json`, right? Why are we calling `set` to update its value? Can't we just mark `view` as a `tracked` property and re-assign via `this.view = view`;

I immediately (and incorrectly) inferred that view was an object, not a string when I read this.

https://guides.emberjs.com/release/in-depth-topics/autotracking-in-depth/#toc_plain-old-javascript-objects-pojos

Reply:
`blocking`:  Should updating this value here, update the query parameter in the controller? This component is being passed this prop from the parent template which has a tracked `view` property as a query parameter. This `view` argument will reactively update to the controller, if we set a change here, shouldn't the controller update?